src/Screens/App/Screens/BooksDashboard/Screens/BookList/index.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import Book from './Screens/Book';
+import books from './Resources/books';
+import './style.css';
+
+class BookList extends Component {
+
+  render() {
+    let check = (book) => {
+      if (this.props.filterType.length != 0 && this.props.filter.length != 0) {
+        switch (this.props.filterType) {
+          case "Author":
+            return book.author.toLowerCase().indexOf(this.props.filter.toLowerCase()) > -1
+          case "Title":
+            return book.title.toLowerCase().indexOf(this.props.filter.toLowerCase()) > -1
```

Comment:
I'd extract this way of filtering in another function.
Another comment is that, we can access to these fields via `[]` operators if `this.props.filterType` is `author` or `title`:
```
return book[this.props.filterType].toLowerCase().indexOf(this.props.filter.toLowerCase()) > -1
```